webserver/views/api/v1/core.py

```diff
@@ -109,3 +111,20 @@ def _validate_offset(offset):
     else:
         offset = 0
     return offset
+
+@bp_core.route("/<user_api_key>/get_pending_jobs", methods=["GET"])
+def get_pending_jobs(user_api_key):
+    """ Return the pending jobs for a user.
+
+    Retrieve the Job Ids for the pending jobs for the user whose
+    user id is given as an argument and return that.
+    """
+    final_result = {}
+    try:
+        pending_jobs = db.dataset_eval.get_user_pending_jobs(user_api_key)
+        for i in range(len(pending_jobs)):
```

Comment:
Why do you want to return this as a dictionary? Is it because `jsonify` won't return a list? In this case, it would be better to wrap the data in a holder object instead. We do this in the dataset module: https://github.com/metabrainz/acousticbrainz-server/blob/78ef1a7051ed8e776b60786d69a51b29e095adad/webserver/views/api/v1/datasets.py#L80
Something like this:
```json
{
 "success": true,
 "datasets": [the,datasets,here]
}
```